cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -14,16 +14,66 @@
  * the License.
  */
 
-function LogViewerController ($scope) {
+function LogViewerController ($scope, LogViewerStore) {
   'ngInject';
 
+  LogViewerStore.subscribe(() => {
+    this.logStartTime = LogViewerStore.getState().startTime;
+    updateTheTable(this.logStartTime);
+  });
+
+  //Use this function to update what is rendered
+  function updateTheTable(value) {
+    console.log('From LogViewerController: ' + value);
```

Comment:
Here too. We should probably remove `console.log` statements. Also if this function is not being used to update the table then we should probably remove the function too.